packages/webapp/socket_file.js

```
@@ -51,11 +55,25 @@ export const removeExistingSocketFile = (socketPath) => {
+    const previousHandlers = registeredHandlers.get(eventEmitter);
+    if (previousHandlers) {
+      for (const [signal, handler] of previousHandlers) {
+        eventEmitter.off(signal, handler);
+      }
+    }
+
+    const handlers = new Map();
     ['exit', 'SIGINT', 'SIGHUP', 'SIGTERM'].forEach(signal => {
-      eventEmitter.on(signal, Meteor.bindEnvironment(() => {
+      const handler = Meteor.bindEnvironment(() => {
         if (existsSync(socketPath)) {
```

ESLint

[error] Delete `··`

(prettier/prettier)